mock-recipe-server/bin/generate.sh

```diff
@@ -3,16 +3,26 @@ set -eu
 
 REPO_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && cd ../.. && pwd)"
 
+compose () {
+    docker-compose \
+        -p mockrecipeserver \
+        -f "$REPO_DIR/compose/docker-compose.yml" \
+        -f "$REPO_DIR/mock-recipe-server/docker-compose.yml" \
+        $@
+}
+
+# Shut down docker even if we error out.
+function finish {
+    compose down
```

Comment:
I think we only need to run `compose stop`. We can't actually remove images in CircleCI, and I think it is probably fine to leave things there but not running.